realm/realm-library/src/objectServer/java/io/realm/SyncCredentials.java

```diff
@@ -156,15 +156,34 @@ public static SyncCredentials custom(String userIdentifier, String identityProvi
      * has logged in. Credentials created this way are automatically assumed to have successfully logged in.
      * This means that providing this credential to {@link SyncUser#login(SyncCredentials, String)} will always
      * succeed, but accessing any Realm after might fail if the token is no longer valid.
+     * <p>
+     * It is assumed that this user is not an administrator. Otherwise use {@link #accessToken(String, String, boolean)}.
      *
      * @param accessToken user's access token.
      * @param identifier user identifier.
      * @return a set of credentials that can be used to log into the Object Server using
      *         {@link SyncUser#loginAsync(SyncCredentials, String, SyncUser.Callback)}
      */
     public static SyncCredentials accessToken(String accessToken, String identifier) {
+        return accessToken(accessToken, identifier, false);
+    }
+
+    /**
+     * Creates credentials from an existing access token. Since an access token is the proof that a user already
+     * has logged in. Credentials created this way are automatically assumed to have successfully logged in.
+     * This means that providing this credential to {@link SyncUser#login(SyncCredentials, String)} will always
+     * succeed, but accessing any Realm after might fail if the token is no longer valid.
+     *
+     * @param accessToken user's access token.
+     * @param identifier user identifier.
+     * @param isAdmin {@code true} if the access token is an administrators token, {@code false} if it is a normal users.
```

Comment:
The boolean is purely for information. The ROS server still has final say, so setting the boolean will purely make some scenarios easier to handle, but I'll update the Javadoc to reflect that you cannot change the state on the server by setting it.